smarts/core/agent_interface.py

```diff
@@ -127,6 +127,8 @@ class AgentType(IntEnum):
     """Controls multiple vehicles"""
     MPCTracker = 10
     """Agent performs trajectory tracking using model predictive control."""
+    TrajectoryInterpolator = 11
+    """Agent performs linear trajectory interpolation."""
     Imitation = 11
```

Comment:
Merge error:  Note that 11 is used by two things now.

Reply:
Oh thanks for letting me know. Will resolve it.

Reply:
Opened PR #978  to fix it.